Codigo/ScenarioKillNpc.cls

```diff
@@ -439,3 +439,65 @@ Private Sub RespawnPlayers()
         End If
     Next key
 End Sub
+
+Sub GiveRewards()
```

Comment:
The method already exists in `Codigo/ScenarioDeathMatch.cls` 

Can we not avoid code duplication moving this to some sort of utility function?

Reply:
yes it is the the exact duplicate from the other one but both of them are in class object and use private data that vb6 doesn't make that easy to share, I might be able to do it but I didn't thing its worth the effort

also this "GiveRewards" functionality will probably be required on every event but the way it shares the reward on the players might change depending the event requierements